src/python/pants/backend/python/goals/pytest_runner.py

```diff
@@ -278,11 +277,21 @@ async def setup_pytest_for_target(
     coverage_args = []
     if test_subsystem.use_coverage and not request.is_debug:
         output_files.append(".coverage")
-        cov_paths = coverage_subsystem.filter if coverage_subsystem.filter else (".",)
+
+        if coverage_subsystem.filter:
+            cov_args = [f"--cov={morf}" for morf in coverage_subsystem.filter]
```

Comment:
The "morf" is Ned's verbage. As I reached back in my memory for this it prompted fixing the --filter help which was lying by omission.